src/Workspaces/Core/Portable/Workspaces.csproj

```diff
@@ -268,6 +268,7 @@
     <InternalsVisibleTo Include="Microsoft.VisualStudio.VisualBasic.Repl" />
     <InternalsVisibleTo Include="Roslyn.VisualStudio.Setup" />
     <InternalsVisibleTo Include="Roslyn.VisualStudio.DiagnosticsWindow" />
+    <InternalsVisibleTo Include="Roslyn.VisualStudio.RemoteHostClientMock" />
```

Comment:
ya. it is testing purpose. I can't put that in diagnostic windows due to all kinds of version conflict between d14 and d15.